src/Compilers/CSharp/Portable/Binder/Binder_Deconstruct.cs

```diff
@@ -16,32 +16,47 @@ namespace Microsoft.CodeAnalysis.CSharp
     /// </summary>
     internal partial class Binder
     {
-        /// <summary>
-        /// Only handles assignment-only or declaration-only deconstructions at this point.
-        /// Issue https://github.com/dotnet/roslyn/issues/15050 tracks allowing mixed deconstructions
-        /// </summary>
         private BoundExpression BindDeconstruction(AssignmentExpressionSyntax node, DiagnosticBag diagnostics)
         {
             var left = node.Left;
             var right = node.Right;
+            CSharpSyntaxNode declaration = null;
```

Comment:
No, they are used as a location for a diagnostic below.

---
In reply to: [90290815](https://github.com/dotnet/roslyn/pull/15548#discussion_r90290815) [](ancestors = 90290815)